src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -152,7 +152,7 @@ def default_api_version(self):
 AZURE_API_PROFILES = {
     'latest': {
         ResourceType.MGMT_STORAGE: '2021-09-01',
-        ResourceType.MGMT_NETWORK: '2021-08-01',
+        ResourceType.MGMT_NETWORK: '2022-01-01',
```

Comment:
Hey, We are planning to get it released with August 2022 release. Why it can't be merged now? Also, when should we able to merge it ?

Reply:
Because all other commands that depend on Network SDK need to re-record their test cases.

I already triggered a pipeline to do it: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1809064&view=results.
As our development close date for current sprint is 08/31, I'll try my best to release it within this milestone (09/06).

Reply:
Hey @necusjz, Can you please share an update on the re-recording the test-cases ? Also, can you please review the PR ?

Reply:
@kumaam We can track the progress by https://github.com/Azure/azure-cli/pull/23709 and this PR can only be merged after the bump is complete.

Actually, there are several test cases need service team help upload recording files and I already contacted them. Hope it can catch up with the current release. But, to be honest, it's still risky.

Reply:
@kumaam 
Updates: I have finished the bump and this PR can be merged after CI passed. Please fix CI according to the log of ADO.

Reply:
Hi,
Thanks for all your support. I've made the changes to fix the CI pipeline.